frontend/src/components/Navbar/index.scss

```
@@ -0,0 +1,50 @@
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 1291px;
+  height: 48px;
+  left: 81px;
+  top: 26px;
```

żeby teraz poprawnie się wyświetlało, trzeba by dodać jeszcze:
```suggestion
  margin: 32px 72px;
  box-sizing: border-box;
```